barretenberg/cpp/src/barretenberg/vm2/common/standard_affine_point.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <cstdint>
+
+namespace bb::avm2 {
+
+template <typename AffinePoint> class StandardAffinePoint {
+  public:
+    using BaseField = AffinePoint::Fq;
+    using ScalarField = AffinePoint::Fr;
+
+    constexpr StandardAffinePoint() noexcept
+        : point(AffinePoint())
+    {}
```

Comment:
`: point(AffinePoint())` is not needed here, it will be default initialized. I'd even try just
`constexpr StandardAffinePoint() noexcept = default;` but it might not fly.